cmd/dump.go

```diff
@@ -175,5 +175,7 @@ configure Kong.`,
 		false, "assume `yes` to prompts and run non-interactively.")
 	dumpCmd.Flags().BoolVar(&dumpConfig.SkipCACerts, "skip-ca-certificates",
 		false, "do not dump CA certificates.")
+	dumpCmd.Flags().BoolVar(&dumpConfig.SkipLicenses, "skip-licenses",
+		true, "do not diff licenses.")
```

Comment:
After we come up with a decision about [this](https://github.com/Kong/deck/pull/882/files#r1171240128), can you also please fix the doc strings? These are all referring to `diff` regardless of the command type